src/mobile/config.coffee

```diff
@@ -67,6 +67,7 @@ module.exports =
   APPLE_TEAM_ID: null
   APPLE_KEY_ID: null
   APPLE_PRIVATE_KEY: null
+  ZENDESK_KEY: null
```

Comment:
I think pages are responsive now so we probably don't need to add the key to mobile config.